docs/tutorials.md

```diff
@@ -2,28 +2,460 @@
 
 @sphinxdirective
 
+:target:`doxid-tutorials_1md_openvino_docs_tutorials`
+
 .. _notebook tutorials:
 
 .. toctree::
    :maxdepth: 2
    :caption: Notebooks
    :hidden:
+   :glob:
 
-   notebooks/notebooks
+   notebooks-installation
+   notebooks/*
 
-@endsphinxdirective
+This collection of Python tutorials are written for running on Jupyter notebooks. 
+The tutorials provide an introduction to the OpenVINO™ toolkit and explain how to 
+use the Python API and tools for optimized deep learning inference. You can run the 
+code one section at a time to see how to integrate your application with OpenVINO 
+libraries.
 
-This collection of Python tutorials are written for running on [Jupyter*](https://jupyter.org) notebooks. The tutorials provide an introduction to the OpenVINO™ toolkit and explain how to use the Python API and tools for optimized deep learning inference. You can run the code one section at a time to see how to integrate your application with OpenVINO™ libraries.
+Notebooks with a |binder logo| button can be run without installing anything. 
```

Comment:
we can also add a sentence here to mention when I need install notebooks and what's the benefit, with a link to the installation page.

Reply:
Hmm, there is a note below, stating that, I guess :blush:

Reply:
Great! Sorry I missed that.